pkg/crc/services/dns/template.go

```diff
@@ -22,7 +22,10 @@ address=/{{ .AppsDomain }}/{{ .IP }}
 address=/etcd-0.{{ .ClusterName}}.{{ .BaseDomain }}/{{ .IP }}
 address=/api.{{ .ClusterName}}.{{ .BaseDomain }}/{{ .IP }}
 address=/api-int.{{ .ClusterName}}.{{ .BaseDomain }}/{{ .IP }}
-address=/{{ .Hostname }}.{{ .ClusterName}}.{{ .BaseDomain }}/{{ .IP }}
+addn-hosts=/etc/hosts.openshift
+`
+	dnsmasqHostsTemplate = `192.168.126.11 {{ .Hostname }}.{{ .ClusterName}}.{{ .BaseDomain }}
+{{ .IP }} {{ .Hostname }}.{{ .ClusterName}}.{{ .BaseDomain }}
```

Comment:
This has to be done in a hosts file, this can't be done through regular DNS?

Reply:
@cfergeau regular dns you mean the dnsmasq config? that is not possible because for same domain if you need different ip you need to use the host file.